From b6c97e3158554f29c1467011e8d50ad2b2350bc1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Piotr=20Pawe=C5=82=20Stefaniak?=
 <3462925+pstef@users.noreply.github.com>
Date: Sat, 11 May 2024 19:12:53 +0200
Subject: [PATCH] Fix build on some systems
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

The complaint was: error: ‘reverse’ is not a member of ‘std’
---
 Source/Core/Core/Debugger/PPCDebugInterface.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Source/Core/Core/Debugger/PPCDebugInterface.cpp b/Source/Core/Core/Debugger/PPCDebugInterface.cpp
index f44c20d7f28c..1ebd0f8d5b48 100644
--- a/Source/Core/Core/Debugger/PPCDebugInterface.cpp
+++ b/Source/Core/Core/Debugger/PPCDebugInterface.cpp
@@ -4,6 +4,7 @@
 
 #include "Core/Debugger/PPCDebugInterface.h"
 
+#include <algorithm>
 #include <array>
 #include <cstddef>
 #include <string>

